src/Symfony/Component/Validator/Tests/Constraints/EmailValidatorTest.php

```diff
@@ -94,23 +93,23 @@ public function getInvalidEmails()
         );
     }
 
-    public function testStrict()
+    public function testRfc()
     {
-        $constraint = new Email(array('strict' => true));
```

Comment:
Some tests must be kept with the existing name, to cover backward compatibility (if this is deprecated, it should be come a legacy test, but it should not disappear)

Reply:
I have a depreciation notice being raised for usage on the `strict` property. However this means the test that checks `strict` still works fails. What's the best thing to do?